lib/ansible/modules/network/netconf/netconf_config.py

```diff
@@ -91,7 +91,7 @@
     version_added: "2.7"
   error_option:
     description:
-    - This option control the netconf server action after a error is occured while editing the configuration.
+    - This option control the netconf server action after a error is occurred while editing the configuration.
```

Comment:
Thanks for correcting the typo. With the spelling correct, the grammar mistakes really stand out. Additional fix:
```suggestion
    - This option controls the netconf server action after an error occurs while editing the configuration.
```

Reply:
yep, i fixed typos mostrly automatically..  out of context
thanks!